Module-1-Github/Day-3/Tasks/hamzah.task

```diff
@@ -0,0 +1,8 @@
+Git revert:
+The git revert command can be considered an 'undo' type command, however, it is not a traditional undo operation. Instead of removing the commit from the project history, it figures out how to invert the changes introduced by the commit and appends a new commit with the resulting inverse content. This prevents Git from losing history, which is important for the integrity of your revision history and for reliable collaboration.
```

Comment:
Update the commands with tracking current HEAD and reverting numbers of commits from head.

Reply:
this have been reverted